src/server/handlers/github.js

```diff
@@ -196,7 +196,7 @@ export const createWebhook = async (req, res) => {
           return res.status(401).send(RESPONSE_AUTHENTICATION_FAILED);
         case 'Validation Failed':
           // Webhook already created
-          return res.status(422).send(RESPONSE_ALREADY_CREATED);
+          return res.status(200).send(RESPONSE_ALREADY_CREATED);
```

Comment:
Because this API endpoint is called by the call-api middleware now, it cannot tell the difference between a 422 "already created" error response and any other error. The call-api middleware is designed to treat all non-2XX errors the same and treat it as a failure.

This suggested to me that surfacing a 4XX error to the client was not the correct approach in the first place, because it does not result in a failure outcome for the application.